bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.h

```diff
@@ -2,7 +2,7 @@
 #include <fcntl.h>
 #include <stdio.h>
 #include <unistd.h>
-#include <string.h>
+#include "string.h"
```

Comment:
This import used angle brackets intentionally to reference the system string utility rather than a local file. [GCC docs on search paths](https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html).

```suggestion
#include <string.h>
```

Reply:
Android Studio seems to automatically suggest this as an import - I'll revert this change.